grammars/supercollider.cson

```diff
@@ -7,7 +7,7 @@
 'name': 'SuperCollider'
 'patterns': [
   {
-    'match': '\\b(arg|var|classvar|this|thisThread|thisMethod|thisFunction|thisProcess|true|false|inf|nil)\\b'
+    'match': '\\b(arg|var|classvar|const|this|thisThread|thisMethod|thisFunction|thisProcess|true|false|inf|nil)\\b'
```

Comment:
Looks like we're missing a few other keywords too — `thisFunction` and `thisFunctionDef` are also built-in references like `thisMethod`. Would you mind adding them?

Reply:
No problem. `thisFunction` was already there but I added `thisFunctionDef` in commit ca48d9b.